free-programming-cheatsheets-en.md

```diff
@@ -0,0 +1,108 @@
+### Index
+
+* [Algorithms & Data Structures](#algorithms--data-structures)
+* [Android](#android)
+* [Angular](#angular)
+* [Bootstrap](#bootstrap)
+* [C](#c)
+* [C#](#c-sharp)
+* [C++](#c-1)
+* [Git](#git)
+* [iOS](#ios)
+* [Java](#java)
+* [JavaScript](#javascript)
+  * [Next.js](#nextjs)
+  * [NodeJS](#nodejs)
+  * [React](#react)
+  * [React Native](#react-native)
+  * [Redux](#redux)
+* [PHP](#php)
+* [Python](#python)
+  * [Django](#django)
+  * [Flask](#flask)
+
+
+### algorithms--data-structures
+* [Algorithms and Data Structures Cheatsheet](https://algs4.cs.princeton.edu/cheatsheet/) - Princeton
+* [Data Structures Cheatsheet](https://intellipaat.com/mediaFiles/2019/02/Python-Data-structures-cheat-sheet.pdf) - Intellipaat
+
+
+### android
+* [Android Cheatsheet](https://github.com/anitaa1990/Android-Cheat-sheet) - Anita Murthy
+
+
+### angular
+* [Angular Cheatsheet](https://angular.io/guide/cheatsheet) - Angular Dev team
+
+
+### Bootstrap
+* [Bootstrap Cheatsheet](https://hackerthemes.com/bootstrap-cheatsheet/) - Hacker Themes
+* [Bootstrap Cheatsheet PDF](https://websitesetup.org/bootstrap-cheat-sheet/) - Website Setup
+
+
+### C
+* [C Cheatsheet](https://courses.cs.washington.edu/courses/cse351/16wi/sections/1/Cheatsheet-c.pdf) - The University of Washington
+
+
+### C#
+* [C# Cheatsheet](https://hackr.io/blog/c-sharp-cheat-sheet) - Hackr
+
+
+### C++
+* [C++ Cheatsheet](https://github.com/mortennobel/cpp-cheatsheet) - Morten Nobel-Jørgensen
+
+
+### Git
+* [Git Cheatsheet](https://www.atlassian.com/git/tutorials/atlassian-git-cheatsheet) - Atlassian
+
+
+### IOS
+* [IOS Cheatsheet](https://koenig-media.raywenderlich.com/uploads/2019/11/RW-Swift-5.1-Cheatsheet-1.0.1.pdf) - raywenderlich.com

```

Comment:
Perhaps use the name of the resource here instead?

Also please append (PDF) if it's a PDF file.

```suggestion
* [Swift 5.1 Cheat Sheet and Quick Reference](https://koenig-media.raywenderlich.com/uploads/2019/11/RW-Swift-5.1-Cheatsheet-1.0.1.pdf) - raywenderlich.com (PDF)
```